gnome-initial-setup/gis-driver.c

```diff
@@ -904,6 +938,26 @@ emit_startup_sound (GApplication * app, gpointer user_data)
                      G_DBUS_CALL_FLAGS_NONE, -1, NULL, NULL, NULL);
 }
 
+static void
+connect_to_gdm (GisDriver *driver)
```

Comment:
I made this accept the `GisDriver` rather than the `Priv` structure: this is more conventional.